package.json

```diff
@@ -23,6 +23,7 @@
     "active-handles": "1.1.0",
     "gc-profiler": "1.3.1",
     "insync": "2.1.1",
+    "joi": "7.2.2",
```

Comment:
This is already in `devDependencies`. I have the server specific stuff in there since it's going to be removed from this repo.